src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,80 @@
+package duke.commands;
+
+import duke.*;
```

Comment:
Instead of wildcard imports, you may want to show every `duke` class you imported. Perhaps you may want to check your IDE settings if need be, sometimes they automatically condense all your imports from the same package if they hit a certain minimum requirement.